guide/src/features.md

```diff
@@ -93,6 +93,15 @@ These features enable conversions between Python types and types from other Rust
 
 Adds a dependency on [anyhow](https://docs.rs/anyhow). Enables a conversion from [anyhow](https://docs.rs/anyhow)’s [`Error`](https://docs.rs/anyhow/latest/anyhow/struct.Error.html) type to [`PyErr`](https://docs.rs/pyo3/latest/pyo3/struct.PyErr.html), for easy error handling.
 
+### `chrono`
+Adds a dependency on [chrono](https://docs.rs/chrono). Enables a conversion from [chrono](https://docs.rs/chrono)'s types to python:
+- [Duration](https://docs.rs/chrono/0.4.22/chrono/struct.Duration.html) -> [`PyDelta`](https://docs.rs/pyo3/0.17.1/pyo3/types/struct.PyDelta.html)
+- [FixedOffset](https://docs.rs/chrono/0.4.22/chrono/offset/struct.FixedOffset.html) -> [`PyDelta`](https://docs.rs/pyo3/0.17.1/pyo3/types/struct.PyDelta.html)
+- [Utc](https://docs.rs/chrono/0.4.22/chrono/offset/struct.Utc.html) -> [`PyTzInfo`](https://docs.rs/pyo3/0.17.1/pyo3/types/struct.PyTzInfo.html)
+- [NaiveDate](https://docs.rs/chrono/0.4.22/chrono/naive/struct.NaiveDate.html) -> [`PyDate`](https://docs.rs/pyo3/0.17.1/pyo3/types/struct.PyDate.html)
+- [NaiveTime](https://docs.rs/chrono/0.4.22/chrono/naive/struct.NaiveTime.html) -> [`PyTime`](https://docs.rs/pyo3/0.17.1/pyo3/types/struct.PyTime.html)
+- [DateTime](https://docs.rs/chrono/0.4.22/chrono/struct.DateTime.html) -> [`PyDateTime`](https://docs.rs/pyo3/0.17.1/pyo3/types/struct.PyDateTime.html)
```

Comment:
I don't think we need to manually link each type here, doing it this way can make the links pointing to the wrong version easily.

Reply:
yes I was not sure about this since pointing to the exact version could lead to outdated documentation if pyo3 upgrades the dependency, but pointing to latest will link to the wrong one if chrono changes anything relevant.

The reason I decided to point to the exact version is that at least pyo3 has control over this file, so it can be upgraded when the dependency is, but maybe just listing the structs without linking them could be enough.

@davidhewitt let me know what you prefer here and I'll fix this